main/webapp/modules/core/langs/translation-en.json

```diff
@@ -58,8 +58,9 @@
     "core-index-create/formats": "TSV, CSV, *SV, Excel (.xls and .xlsx), JSON, XML, RDF as XML, and Google Data documents are all supported. Support for other formats can be added with OpenRefine extensions.",
     "core-index-create/from": "Get data from",
     "core-index-import/import-proj": "Import project",
-    "core-index-import/locate": "Locate an existing Refine project file (.tar or .tar.gz):",
+    "core-index-import/locate": "Locate an existing Refine project file or use a URL (.tar or .tar.gz):",
```

Comment:
Should this have a new id to reflect the fact that the semantics have changed? How do we signal the translators that the existing translation is no longer valid and needs retranslating?

Reply:
That's handled by Weblate automatically. The existing translations are marked as stale and translators are incentivized to update them. In the meantime, the old translation is still shown in the UI, which is fine I think

Reply:
OK, thanks. I guess I was putting too strong a semantic value on the identifier. Sorry for the noise!